sphinxcontrib_images_lightbox2/lightbox2.py

```diff
@@ -23,7 +23,8 @@ def visit_image_node_html(self, writer, node):
                                        builder.images[node['uri']])
 
         writer.body.append(
-            u'''<a data-lightbox="{group}"
+            u'''<figure class="{cls}">
```

Comment:
I think it's better to have the class only in the ``figure`` as it's the top element, but I was afraid to break existing styles by removing the class in the ``a``.
I will remove the class in the ``figcaption``, this one is definitely useless.